pkg/features/kube_features.go

```diff
@@ -639,6 +639,7 @@ const (
 
 	// owner: @zshihang
 	// alpha: v1.20
+	// beta: v1.21
```

Comment:
A couple missing things:
- Update the comments in types.go indicating the API fields are beta.
- Update the e2e test to remove the "Feature" tag.

Reply:
done. thanks.